source/gatherings/template.html.erb

```diff
@@ -90,6 +90,24 @@ description:  The OpenShift Commons community gets together and share experience
   </div><!-- /.row -->
 </div><!-- /.container-->
 <br>
+<!-- -->
+<section id="gathering-over-notice">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12 text-center">
+        <p>
+          <strong>This event is now over</strong>!
+          <% if gathering.youtube_playlist_id.presence && gathering.youtube_playlist_id.length > 0 %>
+            All of the presenations were recorded and are available on
+            <a href="https://www.youtube.com/channel/UCZKMj3YI0wP-kq4QYpaKdEA" target="_blank">our Youtube channel</a> in
+            <a href="https://www.youtube.com/playlist?list=<%= gathering.youtube_playlist_id %>" target="_blank">
+            <strong>this playlist</strong></a>.
+          <% end %>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
```

Comment:
Maybe put this section at the top of the template? This way the notice is "above the fold" when visiting.

Reply:
I've requested review from @dmueller2001 too. I personally prefer the notice closer to the event overview with the actual calendar icon, but it sure can be even on more places... e.g. on top and below the event overview (not to be too close to each other, while still have some visual confirmation why the calendar is gray and links not working close to the overview section). With the design touches, I'll wait for additional @dmueller2001 comments.